src/resources/auditlog.js

```diff
@@ -0,0 +1,21 @@
+import List from '../actions/list';
+import Find from '../actions/find';
+import Resource from './base';
+
+export default class AuditLog extends Resource {
+  get resourceName() {
+    return 'audit_log';
+  }
+
+  get resourcesName() {
+    return 'audit_logs';
+  }
+
+  get statement() {
+    console.log('Statement from the Audit Log object');
```

Comment:
Do we plan to implement this method? This might be a better fit for fulcrumapp/fulcrum-core

Reply:
oh whoops that is a logging function that I accidentally left in